x-pack/filebeat/module/checkpoint/firewall/ingest/pipeline.yml

```
@@ -40,10 +40,14 @@ processors:
     - message
     - host
     ignore_missing: true
-- set:
-    field: '@timestamp'
-    value: '{{syslog5424_ts}}'
-    if: ctx.checkpoint?.time == null
+- rename:
+    field": "@timestamp"
+    target_field: "event.created"
+    if: "ctx.@timestamp != null"
+- date:
+    field: "syslog5424_ts"
+    formats: ["ISO8601"]
```

```suggestion
    formats: ["ISO8601", "UNIX"]
```

not sure if this can come as unix at some point also